zowe-rest-api-sample-kotlin-spring/src/main/resources/messages_en.properties

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
No entries, why do we have this file?

Reply:
This is a different solution to language issues.

Issue without this file:
The client sends an English request. There are only the default messages without language, Czech, and Spanish on the server. No English language is found so the server language is used. When it is English, It is not found as well and the default file is used. It is written in English so it looks okay. When the server is running with the Czech language, the Czech is found and the Czech response is returned. This is bad because you were asking for English.

Solution: Add the English message file. When you ask for English and the language file exists, it is used and does not matter on the server. You don't have to replicate the default file which is written also in English because language-specific files inherit the messages from the default one.

I hope it is somehow understandable.